src/d1to2fix/symbolsearch.d

```diff
@@ -38,7 +38,10 @@ public void initializeModuleCache(string[] paths)
     import std.exception;
     enforce(delegate_cache is null);
 
-    auto module_cache = new ModuleCache(new ASTAllocator);
+    // static is used so that pointer persists and GC won't collect it
+    // together with underlying owned dsymbols
```

Comment:
No, this is "work as intended" domain of issues. Most of EMSI containers implement owning semantics and manually allocate memory internally. It is not designed to be used with GC like in old code.